app/components/employeeCV.tsx

```diff
@@ -0,0 +1,84 @@
+"use client";
+import React, { useEffect, useState } from "react";
+import { EmployeeCV } from "../function/Employees";
+import { ErrorBoundary } from "./error";
+import dynamic from "next/dynamic";
+import { Loading } from "./home";
+import { useChatStore } from "../store";
+import { isMobileScreen } from "../utils";
+import styles from "../components/home.module.scss";
+import Sidebar from "./sidebar";
+import SalesGPTPrompt from "./salesGPTPropmt";
+import CV from "./CV";
+
+const Settings = dynamic(async () => (await import("./settings")).Settings, {
+  loading: () => <Loading noLogo />,
+});
+
+const Employees = dynamic(async () => await import("./employees"), {
+  loading: () => <Loading noLogo />,
```

Comment:
Foreslår å sjekke ut bruk av Suspense for loading: https://nextjs.org/docs/app/building-your-application/routing/loading-ui-and-streaming#streaming-with-suspense

Reply:
Dette tipset er nok godt over frontend-kompetansen og ikke heeelt alignet med det vi skal fokusere på atm.

Linker til issue i boardet vårt: https://github.com/orgs/varianter/projects/19?pane=issue&itemId=42057151 som kan plukke opp tråden senere.